testing/environments/docker/kafka/Dockerfile

```diff
@@ -11,7 +11,7 @@ ENV TERM=linux
 RUN apt-get update && apt-get install -y curl openjdk-8-jre-headless netcat
 
 RUN mkdir -p ${KAFKA_LOGS_DIR} && mkdir -p ${KAFKA_HOME} && curl -s -o $INSTALL_DIR/kafka.tgz \
-    "http://ftp.wayne.edu/apache/kafka/${KAFKA_VERSION}/kafka_2.11-${KAFKA_VERSION}.tgz" && \
+    "http://mirror.easyname.ch/apache/kafka/${KAFKA_VERSION}/kafka_2.11-${KAFKA_VERSION}.tgz" && \
```

Comment:
makes me wonder why we do not download from archive.apache.org

Reply:
Good question, probably because it was once down in the past and we switched it? :-) I think they always recommend local mirrors?

Reply:
`.ch` :)

Reply:
yeah, well. I fall into the trap of what it recommends me and worked locally :-D

Reply:
i see a trend of 2.1.0 disappearing from mirrors, maybe we can consider bumping up patch number instead of looking for mirror with slowest refresh rate

Reply:
@michalpristas SGTM. Could you take this on?